erpnext/accounts/party.py

```diff
@@ -506,12 +517,18 @@ def get_dashboard_info(party_type, party):
 
 		total_unpaid = flt(company_wise_total_unpaid.get(d.company))
 
+		if loyalty_point_details:
```

Comment:
Initialize loyalty_point_details as an empty list before `if party_type == "Customer":` Line No: 482.

Without this for other party types, this will result as

<img width="456" alt="screen shot 2018-12-26 at 10 35 15 am" src="https://user-images.githubusercontent.com/3784093/50432808-3d038d80-08fa-11e9-99bc-b3587fca362a.png">

Reply:
@saurabh6790  Done